CreeDictionary/CreeDictionary/views.py

```diff
@@ -46,7 +64,7 @@ def search_results(request, query_string: str):  # pragma: no cover
     )
 
 
-def lemma_details(request, lemma_id: int):  # pragma: no cover
+def _lemma_details(request, lemma_id: int):  # pragma: no cover
```

Comment:
Could you please rename this to something like `lemma_details_internal` or something like that? Basically, a name that makes it clear that it's for internal use only and not a traditional view.

Reply:
I will